documentation/modules/ROOT/pages/configuration/signalling.adoc

```diff
@@ -43,6 +43,14 @@ You can specify which channel is enabled by setting the `signal.enabled.channels
 The `source` channel is enabled by default, because it is required for incremental snapshot signals.
 
 
+
+=== Error handling
+
+All channels, except for the `source`, doesn't have any retry policy.
+This means that is up to you to monitor whenever a signal completed correctly or not.
```

Comment:
```suggestion
With the exception of the `source` channel, {prodname} signaling channels do not implement retry policies.
After you initiate a signal, be sure to verify whether it completes successfully.
```
Is there any specific external automatic monitoring mechanism that a user could implement?